### ISSUE TITLE
Tab completion with aliases broken in v0.64.0

### ISSUE THREAD

**Issue by lukexor** [nushell/nushell#5802]
### Describe the bug

All kinds of completions seem broken as of the latest version. Was last working fine in 0.62 I believe.

### How to reproduce

`alias ll=ls -l`
```
[19:44] v0.64.0 ≡ ll bin<TAB>
```

turns into

```
[19:47] v0.64.0 ❯ ll b-l
```

### Expected behavior

Expect `alias ll=ls -l` to have similar directory tab completion to plain `ls`

### Configuration

| key                | value                               |
| ------------------ | ----------------------------------- |
| version            | 0.64.0                              |
| branch             | main                                |
| commit_hash        |                                     |
| build_os           | macos-x86_64                        |
| build_target       | x86_64-apple-darwin                 |
| rust_version       | rustc 1.61.0 (fe5b13d68 2022-05-18) |
| rust_channel       | stable-x86_64-apple-darwin          |
| cargo_version      | cargo 1.61.0 (a028ae42f 2022-04-29) |
| pkg_version        | 0.64.0                              |
| build_time         | 2022-06-15 10:03:28 -07:00          |
| build_rust_channel | release                             |
| features           | default, trash, which, zip          |
| installed_plugins  |                                     |

### Additional context

Maybe not related, but
```
[19:48] v0.64.0 ≡ ls <TAB>
n                                   Switch to the next shell.
x                                   Extract a packaged file.
g                                   Switch to a given shell.
p                                   Switch to the previous shell.
```

What is that tab completion even trying to suggest? nushell knows about `ls` so it should offer file completion.

My `tab` mapping:
```
    {
      name: completion_menu
      modifier: none
      keycode: tab
      mode: vi_insert
      event: {
        until: [
          { send: menu name: completion_menu }
          { send: menunext }
        ]
      }
    }
```

**Comment by Kangaxx-0** [comment]
I have a draft to fix this issue, however, the `Additional context` in descption is a different issue and not covered at this time

**Comment by WindSoilder** [comment]
> the Additional context in descption is a different issue and not covered at this time

I think it's fixed in #5805

**Comment by adamshackleford** [comment]
It's not just aliases. In 0.61.0, typing a subdirectory name, then pressing Tab would append "./" to the front, so you could press Enter and cd to the subdirectory. 

In 0.64.0 (and maybe earlier), pressing Tab just gives you a trailing "/". Pressing Enter at this point gives "can't run executable".

**Comment by icp1994** [comment]
Seems to be fixed for me on `v0.65` :clinking_glasses:

**Comment by adamshackleford** [comment]
Ah, good news.

**Comment by hustcer** [comment]
> Seems to be fixed for me on `v0.65` 🥂


Then let's close it, feel free to reopen if it doesn't work
